planner/core/rule_join_reorder_greedy.go

```diff
@@ -101,6 +103,7 @@ func (s *joinReorderGreedySolver) constructConnectedJoinTree() (*jrNode, error)
 			p:       bestJoin,
 			cumCost: bestCost,
 		}
+		tracer.appendLogicalJoinCost(bestJoin, bestCost)
```

Comment:
Can we have test cases to cover the cost of joins traced here?

Reply:
updated